### DISCUSSION TITLE
Weird secondary explosion with gravity railgun shells in test mode

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#14975] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Some weird visual explosion happening after and away from the point of impact/gravity pull

Evil Factory added some additional info when I mentioned it in discord

Evil Factory : 
theres also a bug report that the gravity shells don't get properly removed and instead get teleported to 0,0
I think it can only be effectively seen in test mode, since its the only one where the sub is on 0,0 by default

### Reproduction steps

1. Humpback test mode
2. Spawn gravity railgun shell
3. lock y (optional), disableenemyai and spawn Bonethreshers
4. Shoot at Bonethreshers

https://github.com/user-attachments/assets/89c6ad5c-07b2-47c2-8f0c-f474b055f219

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in a future update.
